### ISSUE TITLE
Exception error in fresh installed owncloud 9

### ISSUE THREAD

**Issue by Liwindo** [owncloud/core#23099]
### Steps to reproduce
1. Install owncloud 9.0.0

### Expected behaviour
Everything works fine

### Actual behaviour
Exception error, no apps are found and files can not be uploaded.

### Server configuration
**Operating system**: Ubuntu 14.04.4

**Web server:** Apache with nginx proxy

**Database:** MySQL 5.5.47

**PHP version:** 7.0.3

**ownCloud version:** 9.0.0.19

**Updated from an older ownCloud or fresh install:** fresh install

Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here: No errors have been found.

**The content of config/config.php:**

<?php
$CONFIG = array (
  'instanceid' => 'SECRET',
  'passwordsalt' => 'SECRET',
  'secret' => 'SECRET',
  'trusted_domains' => 
  array (
    0 => 'domain.tld',
  ),
  'datadirectory' => '/path_to_owncloud/data',
  'overwrite.cli.url' => 'https://domain.tld',
  'dbtype' => 'mysql',
  'version' => '9.0.0.19',
  'dbname' => 'db_name',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'db_user',
  'dbpassword' => 'password',
  'logtimezone' => 'UTC',
  'installed' => true,
);

**Are you using encryption:** not possible to activate

**Are you using an external user-backend, if yes which one:** no

#### ownCloud log (data/owncloud.log)
```
Error 	PHP 	Undefined index: newVersionString at /path_to owncloud/apps/updatenotification/templates/admin.php#8 	2016-03-10T14:19:38+00:00
Error 	no app in context 	Exception: {"Exception":"DomainException","Message":"Calendar tables are missing. Nothing to do.","Code":0,"Trace":"#0 \/path_to_owncloud\/apps\/dav\/lib\/migration\/migratecalendars.php(84): OCA\\Dav\\Migration\\CalendarAdapter->setup()\n#1 \/path_to_owncloud\/apps\/dav\/appinfo\/application.php(202): OCA\\Dav\\Migration\\MigrateCalendars->setup()\n#2 \/path_to_owncloud\/apps\/dav\/appinfo\/install.php(27): OCA\\Dav\\AppInfo\\Application->migrateCalendars()\n#3 \/path_to_owncloud\/lib\/private\/installer.php(621): include('\/var\/www\/vhosts...')\n#4 \/path_to_owncloud\/lib\/private\/installer.php(572): OC_Installer::includeAppScript('\/var\/www\/vhosts...')\n#5 \/path_to_owncloud\/lib\/private\/installer.php(546): OC_Installer::installShippedApp('dav')\n#6 \/path_to_owncloud\/lib\/private\/setup.php(370): OC_Installer::installShippedApps()\n#7 \/path_to_owncloud\/core\/controller\/setupcontroller.php(64): OC\\Setup->install(Array)\n#8 \/path_to_owncloud\/lib\/base.php(823): OC\\Core\\Controller\\SetupController->run(Array)\n#9 \/path_to_owncloud\/index.php(39): OC::handleRequest()\n#10 {main}","File":"\/path_to_owncloud\/apps\/dav\/lib\/migration\/calendaradapter.php","Line":68} 
```

**Comment by DeepDiver1975** [comment]
Duplicate - this was already reported and will be fixed with 9.0.1

**Comment by fatbearinc** [comment]
Is there a workaround that we can apply now or is the present production version of ownCloud (CentOS 7.2: owncloud-files-9.0.0-1.1.noarch) that is installed using the [repo](http://download.owncloud.org/download/repositories/stable/CentOS_7/ce:stable.repo) dead on arrival?
